### DISCUSSION TITLE
Position, Font, Size, Color

### DISCUSSION THREAD

**Discussion by Nightscars** [Araxeus/Youtube-Volume-Scroll#10] [Q&A] [answered]
Thank you so much for creating this first off. It's surprisingly hard to find a extension that changes the volume of YouTube videos and I'm surprised that google has not incorporated something so simple.

I was wondering if it would be possible to add the ability to change the position of the volume HUD %. Like upper right, upper left, bottom left or bottom right. Along with changing the size, font and color of the volume HUD % would be awesome as well.
Not sure if that's a thing that can be add or is doable.

**Comment by Araxeus** [comment]
Thank you for your feedback, I originally made this extension for private use but then decided other people should enjoy it too

----

> I was wondering if it would be possible to add the ability to change the position of the volume HUD %. Like upper right, upper left, bottom left or bottom right. Along with changing the size, font and color of the volume HUD % would be awesome as well.
Not sure if that's a thing that can be add or is doable.

All of that is doable but adding all those options might bloat the extension popup

do you have a suggestion on how it could be styled? (I'm a pretty bad designer 😅)

I have a crude mock drawing to start:

![Untitled](https://user-images.githubusercontent.com/78568641/180241734-fa540fbc-993b-476f-a7d6-265e1e625577.png)

**Reply by Nightscars** [reply]
Hello,
Sorry for the late reply.. RL and all that.
I'm not a very well designer either. However the simpler the better imo.
What you have drawn up would work perfectly fine for the use case.
Thank you again for sharing this plugin. I have gotten so used to scrolling my mouse wheel for YouTube volume I cant imagine not having it lol.

**Comment by Olizja** [comment]
I use version 1.2.1 and corrected it a little. https://i.imgur.com/rJDb4hZ.png
When will you do what the person who created this question asked?

**Reply by Araxeus** [reply]
I'm building up the mental courage to do it 😅 

I'll get to it eventually, sorry

(btw you're welcome to contribute if you want)

**Comment by Araxeus** [comment, marked as answer]
Good news, this was implemented in #29 and a new version has been published https://github.com/Araxeus/Youtube-Volume-Scroll/releases/tag/v3.0.0

![image](https://user-images.githubusercontent.com/78568641/218888746-4ace1132-fc6a-4e4b-8443-c9a964a1bef4.png) | ![image](https://user-images.githubusercontent.com/78568641/218888909-8b458a73-7046-4cc6-9e89-f5181b6cf017.png)
|:-:|:-:|
